.changeset/stinky-cars-fart.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
```

Comment:
Can you add a concise description of the change? Examples can be found in #22965